src/Symfony/Component/Asset/VersionStrategy/JsonManifestVersionStrategy.php

```diff
@@ -53,13 +59,24 @@ public function applyVersion(string $path)
     private function getManifestPath(string $path): ?string
     {
         if (null === $this->manifestData) {
-            if (!is_file($this->manifestPath)) {
-                throw new \RuntimeException(sprintf('Asset manifest file "%s" does not exist.', $this->manifestPath));
-            }
+            if (0 === strpos(parse_url($this->manifestPath, \PHP_URL_SCHEME), 'http')) {
+                $httpclient = $this->httpClient ?? HttpClient::create();
```

Comment:
Maybe we shouldn't do this ` ?? HttpClient::create()` fallback. That would give more control to users than want to be sure that the manifest is local.